src/wpapsk_fmt_plug.c

```diff
@@ -212,33 +210,27 @@ struct fmt_main fmt_wpapsk = {
 		get_binary,
 		get_salt,
 		{
-			get_keyver,
+			NULL //get_keyver,
 		},
 		fmt_default_source,
 		{
-			fmt_default_binary_hash_0,
-			fmt_default_binary_hash_1,
-			fmt_default_binary_hash_2,
-			fmt_default_binary_hash_3,
-			fmt_default_binary_hash_4,
-			fmt_default_binary_hash_5,
-			fmt_default_binary_hash_6
+			binary_hash_0,
+			binary_hash_1,
+			binary_hash_2,
+			binary_hash_3,
+			binary_hash_4,
+			binary_hash_5,
+			binary_hash_6
 		},
 		salt_hash,
-		salt_compare,
+		NULL, //salt_compare,
 		set_salt,
 		set_key,
 		get_key,
-		clear_keys,
+		fmt_default_clear_keys,
 		crypt_all,
 		{
-			get_hash_0,
-			get_hash_1,
-			get_hash_2,
-			get_hash_3,
-			get_hash_4,
-			get_hash_5,
-			get_hash_6
+			fmt_default_get_hash,
```

Comment:
Looks wrong to me: why use 7 separate `binary_hash_*` functions when we also use `fmt_default_get_hash`, which always returns 0? And, shouldn't there also be hunks removing the now-unused functions `clear_keys` and `get_hash_*`?

Reply:
With my current hacks (everything is experimental at this point) these are only for **loader** to process pot file entries quicker (it did get WAY quicker). I added code to ignore these functions in cracker when there's no corresponding `get_hash` functions. Not sure if all sizes are required, I just threw them in there.

Yes, there is unused code. It will all be cleaned up before we're anywhere near merging. I'm mostly interested in any caveats you might see with this non-hash scheme.

Reply:
BTW `clear_keys` and `get_hash_*` are (and were) removed, but they were sitting in wpapsk.h